pkg/morph/deploy/deploy.go

```diff
@@ -209,7 +233,32 @@ func Deploy(ctx context.Context, prm Prm) error {
 
 	prm.Logger.Info("committee group successfully initialized", zap.Stringer("public key", committeeGroupKey.PublicKey()))
 
-	// TODO: deploy contracts
+	prm.Logger.Info("updating on-chain NNS contract...")
+
+	err = updateNNSContract(ctx, updateNNSContractPrm{
+		logger:            prm.Logger,
+		blockchain:        prm.Blockchain,
+		monitor:           monitor,
+		localAcc:          prm.LocalAccount,
+		localNEF:          prm.NNS.Common.NEF,
+		localManifest:     prm.NNS.Common.Manifest,
+		systemEmail:       prm.NNS.SystemEmail,
+		committee:         committee,
+		committeeGroupKey: committeeGroupKey,
+		buildVersionedExtraUpdateArgs: func(versionOnChain contractVersion) ([]interface{}, error) {
```

Comment:
but why is it defined here? it does not close anything. why not just a separate func?

Reply:
to be clear, function body may change over time. Guess u suggest to make a global function `func noExtraUpdateArgs()` for this? ok i dont mind

Reply:
> Guess u suggest to make a global function func noExtraUpdateArgs() for this? ok i dont mind

right you are